amida-tech/amida-messaging-microservice
# allow use of RSA for JWT

#### What's this PR do?
Allows use of an RSA public key to verify JWTs generated by the auth service.

Merged in auth service: https://github.com/amida-tech/amida-auth-microservice/pull/10

#### Related JIRA tickets:
[SER-199](https://jira.amida-tech.com/browse/SER-199) / [SER-195](https://jira.amida.com/browse/SER-195)
Duplicate: [SER-235](https://jira.amida.com/browse/SER-235)

#### How should this be manually tested?
- Make sure that you can run the test harness locally for the auth service using RSA mode.
- Set the public key env var for the messaging service to point to the public key used by the auth service.
- Make sure you can run the test harness for the messaging service with RSA mode on.
- If possible, run some simple messaging service workflows. However, using RSA mode for the auth service will break functionality of other services.
